spec/factories/users.rb

```diff
@@ -0,0 +1,24 @@
+include RandomData
+
+FactoryGirl.define do
+  pw = RandomData.random_sentence
+  factory :user do
+    name RandomData.random_name
+    sequence(:email){|n| "user_#{n}@factory.com" }
+    password_digest pw
+    username RandomData.random_sentence
```

Comment:
Interesting approach.  

Why did you choose this over a library like faker?  I'll also often use a sequence like you did with the e-mail ie `sequence(:username) {|n| "user_#{n}" }`

No need to change anything here.. Im just curious to hear more about why you went with this.